community/sharing.md

```diff
@@ -13,6 +16,7 @@ Putting a comment at the top of your code helps people understand it, use it, an
  * https://forum.vivaldi.net/topic/<topic>
 	- - -   
  * Description:        <description>
+ <If your description is long, put the second part here>
```

Comment:
```plaintext
 <If your description is long, put the second part here>
 not here
 *                    but here
```

Reply:
Isn't the (*) for a new line?

Reply:
I mean a new piece of content

Reply:
No, the asterisk if for *any* new line inside the comment.

Reply:
```plaintext
 * I mean, not even here
 *                    but here
```